applicationconfig/client/src/main/java/com/azure/applicationconfig/models/SettingSelector.java

```diff
@@ -175,4 +176,23 @@ public SettingSelector fields(SettingFields... fields) {
         this.fields = fields;
         return this;
     }
+
+    /**
+     * Gets the {@link Range} used to select a specific range of revisions with {@code listSettingRevisions}.
+     * If {@code null}, the service returns all revisions.
+     * @return The {@link Range} used to select a range of revisions.
+     */
+    public Range range() {
+        return (range == null) ? null : new Range(range.start(), range.end());
```

Comment:
Was there an error when we tried to return the range object itself? I wonder why we had to construct a new one if Range is an immutable object.

Reply:
Changed to return just the range value